README.md

```diff
@@ -0,0 +1,96 @@
+# UI-KIOSK
+
+![GitHub package.json version](https://img.shields.io/github/package-json/v/UniversityOfGdanskProjects/UI-KIOSK?style=for-the-badge)
+![GitHub top language](https://img.shields.io/github/languages/top/UniversityOfGdanskProjects/UI-KIOSK?logo=typescript&style=for-the-badge)
```

Comment:
I think I would remove the typescript % badge

Reply:
resolved